src/webgpu/api/validation/gpu_external_texture_expiration.spec.ts

```diff
@@ -187,35 +188,29 @@ g.test('import_and_use_in_different_task')
     });
 
     await waitForNextTask(() => {
-      // Enter in another task scope, previous GPUExternalTexture should be expired.
-      t.submitCommandBuffer(bindGroup, false);
+      // Enter in another task scope. For GPUExternalTexture imported from WebCodec,
+      // it shouldn't be expired because VideoFrame is not 'closed'.
+      // For GPUExternalTexutre imported from HTMLVideoElement, it should be expired.
+      t.submitCommandBuffer(bindGroup, sourceType === 'VideoFrame' ? true : false);
     });
   });
 
 g.test('use_import_to_refresh')
   .desc(
     `
     Tests that in the different task scope, imported GPUExternalTexture
-    again on the same source frame should return the same GPUExternalTexture
+    again on the same HTMLVideoElement should return the same GPUExternalTexture
     object and refresh it.
```

Comment:
Why isn't this test valid for VideoFrame? Even though auto-expiry doesn't happen when importing from VideoFrame, I think the spec still allows importExternalTexture to return the same GPUExternalTexture again.

That said, in no case does the spec actually guarantee that the same GPUExternalTexture will be returned twice. The browser is always allowed to return a new one. So I think this test is stricter than the spec (though perhaps this is OK until someone complains about it).

Reply:
>That said, in no case does the spec actually guarantee that the same GPUExternalTexture will be returned twice. 

That's true. I think I should move the stricter case to chromium content test and release the condition of this cts. That says, the test should test GPUExternalTexture returned by import is active in any task scope.